pkg/controller/instance/instance_controller.go

```diff
@@ -24,6 +24,8 @@ import (
 	"reflect"
 	"time"
 
+	"github.com/kudobuilder/kudo/pkg/kubernetes/status"
```

Comment:
Nit: Move this package to the block of local packages. `goimports` isn't reliable in fixing this automatically :(